src/AutodeskAPS/README.md

```diff
@@ -18,6 +18,7 @@ Please see the [Base Installation Guide](https://socialiteproviders.com/usage/),
   'client_id' => env('AUTODESK_APS_CLIENT_ID'),  
   'client_secret' => env('AUTODESK_APS_CLIENT_SECRET'),  
   'redirect' => env('AUTODESK_APS_REDIRECT_URI'),
+  'region' => env('AUTODESK_APS_REGION'), // Could add a default of EU or US
```

Comment:
Where is it used?

Reply:
@lucasmichot sorry for the late response. It isn't used in the provider right now. But for additional Autodesk API requests it's often required to specify a region.